### PR TITLE
Prettyfication of transmission_main_example

### PR DESCRIPTION
**Pull request by jktjkt**
Esther suggested to create a small animation showing how the tool works in action. Creating a simple screencast led me to make the UI a bit easier to follow. I'm open to suggestions on what to highlight, what to remove, etc., but I think that this PR represents a positive change overall :)

### CODE REVIEW

File: examples/transmission_main_example.py [hunk 1]
```diff
@@ -144,16 +144,15 @@ def main(network, equipment, source, destination, req = None):
...
+            print(f'\nPropagating in {ansi_escapes.cyan}gain mode{ansi_escapes.reset}: power cannot be set manually')
         infos = propagate2(path, req, equipment, show=len(power_range)==1)
         if power_mode:
-            print(f'\nTransmission result for input power = {lin2db(req.power*1e3):.2f}dBm :')
+            print(f'\nTransmission result for input power = {lin2db(req.power*1e3):.2f} dBm:')
         else:
             print(f'\nTransmission results:')
-        #info message in gain mode
-        print(destination)
+        print(f'SNR total (at signal bandwidth): {ansi_escapes.cyan}{mean(destination.snr):.02f} dBm{ansi_escapes.reset}')
```
Comment (ojnas) [inline examples/transmission_main_example.py]: Unit for SNR is dB, not dBm.

Comment (jktjkt) [reply examples/transmission_main_example.py]: Doh, thanks, fixed. I also updated to captured images (thankfully, it was a text-based format so I didn't have to redo it).

**Comment by ojnas** [comment]
I think it's good to print not just total SNR in signal BW but also OSNR ASE and in 0.1nm but see also issue #262.

**Comment by jktjkt** [comment]
I think that #299 is better now.

**Pull request closed without merging**